src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -545,16 +545,24 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
       *(application.get()), resumption, need_restore_vr);
   SendResponse(true, result_code, add_info.c_str(), &response_params);
 
-  if (result_code != mobile_apis::Result::RESUME_FAILED) {
-    resumer.StartResumption(application, hash_id);
+  // Check if application exists, because application might be unregestered
+  // during sending reponse to mobile.
+  application = application_manager_.application(key);
```

Comment:
@pvvasilev , we will have meeting about it in the near future. now local const vars are naming without `k`